### PR TITLE
python 3.6.5

### PR DESCRIPTION
**Pull request by chuanconggao**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/python.rb [hunk 1]
```diff
@@ -1,13 +1,11 @@
...
   homepage "https://www.python.org/"
-  url "https://www.python.org/ftp/python/3.6.4/Python-3.6.4.tar.xz"
-  sha256 "159b932bf56aeaa76fd66e7420522d8c8853d486b8567c459b84fe2ed13bcaba"
-  revision 4
+  url "https://www.python.org/ftp/python/3.6.5/Python-3.6.5.tar.xz"
+  sha256 "f434053ba1b5c8a5cc597e966ead3c5143012af827fd3f0697d21450bb8d87a6"
   head "https://github.com/python/cpython.git"
 
   bottle do
-    rebuild 1
```
Comment (commitay) [inline Formula/python.rb]: The bottle block should not be edited manually.

Comment (chuanconggao) [reply Formula/python.rb]: Thanks for the quick revision.

**Comment by ilovezfs** [comment]
pipenv, pre-commit, sip, vim, and vim@7.4 need revision bumps

**Comment by ilovezfs** [comment]
Thanks!

**Pull request closed without merging**